src/components/settings/presets/ShipPresets.ts

```diff
@@ -0,0 +1,74 @@
+import { SelectedDistributor, SelectedWeapons, WeaponType } from '../../../typescript/data/dataFormat';
+
+// The below import is unused be necessary due to a Svelte issue with enums, described here:
+// https://www.reddit.com/r/sveltejs/comments/n87r2n/getting_a_typescript_enum_in_a_standalone_file_to/
+import { SettingsTabs } from '../SettingTabs';
+
+const empty: ShipPreset = {
```

Comment:
So we don't need this here. Could we just call the empty store functions when this empty button is pressed rather than having something here?

Reply:
Fixed